.github/workflows/manual-fellow-runtimes.yml

```diff
@@ -24,64 +28,105 @@ on:
     - cron: "00 04 * * 1"
 
 jobs:
+  find-runtimes:
+    name: Scan repo ${{ inputs.repo }} ${{ inputs.ref }}
+    outputs:
+      runtime: ${{ steps.get_runtimes_list.outputs.runtime }}
+      commit_hash: ${{ steps.get_commit_hash.outputs.commit_hash }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the srtool repo
+        uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+        with:
+          fetch-depth: 0
+          path: srtool
+
+      - name: Checkout repo ${{ inputs.repo }}
+        uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+        with:
+          repository: ${{ inputs.repo }}
+          ref: ${{ inputs.ref }}
+          fetch-depth: 0
+          path: fellows-runtimes
+
+      - name: Get commit hash for ${{ inputs.repo }} ${{ inputs.ref }}
+        id: get_commit_hash
+        working-directory: fellows-runtimes
+        run: |
+          echo "commit_hash=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
+
+      - name: Cache runtimes list for ${{ steps.get_commit_hash.outputs.commit_hash }}
+        id: cache_runtimes_list
+        if: inputs.cache == 'true'
+        uses: actions/cache@v3
+        with:
+          key: runtimes-list-${{ steps.get_commit_hash.outputs.commit_hash }}
+          path: |
+            fellows-runtimes/runtimes_list.json
+
+      - name: Install tooling
+        if: ${{ steps.cache_runtimes_list.outputs.cache-hit != 'true' }}
+        run: |
+          TOML_CLI_VERSION=0.2.4
```

Comment:
We have ENV at the top, that's a good place to keep such information together.
I would suggest moving that one up with  `SUBWASM_VERSION`.